src/components/input-number/input-number.tsx

```diff
@@ -808,7 +808,7 @@ export class InputNumber
     };
 
     const sanitizedValue = sanitizeNumberString(
-        // no need to delocalize a string that ia already in latn numerals
+      // no need to delocalize a string that ia already in latn numerals
```

Comment:
Was formatted by the linter?

Typo (unrelated): `ia` -> `is`

Reply:
Yes it is.